File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/vssdkdesignerstatusbar/vb/guids.vb
Imports System

Class GuidList
    Private Sub New()
    End Sub

    Public Const guidVSSDKDesignerStatusBarPkgString As String = "825da4a9-45b6-4cd2-a043-c6edddfb1c62"
    Public Const guidVSSDKDesignerStatusBarCmdSetString As String = "d0b75f09-854f-4892-867d-c7147572cabf"

    Public Shared ReadOnly guidVSSDKDesignerStatusBarCmdSet As New Guid(guidVSSDKDesignerStatusBarCmdSetString)
End Class

File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/vssdkdesignerstatusbar/cs/guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Microsoft.VSSDKDesignerStatusBar
{
    static class GuidList
    {
        public const string guidVSSDKDesignerStatusBarPkgString = "38fbbf94-0348-4c53-a05d-54e5eeac95d5";
        public const string guidVSSDKDesignerStatusBarCmdSetString = "0ab16ec6-90c8-4af2-9efc-59d79feb1886";

        public static readonly Guid guidVSSDKDesignerStatusBarCmdSet = new Guid(guidVSSDKDesignerStatusBarCmdSetString);
    };
}